localstack/services/iam/iam_listener.py

```diff
@@ -1,62 +1,25 @@
-import re
 from requests.models import Request
-from localstack.constants import TEST_AWS_ACCOUNT_ID, MOTO_ACCOUNT_ID
-from localstack.utils.aws import aws_stack
-from localstack.utils.common import to_str
 from localstack.services.generic_proxy import ProxyListener
+from localstack.utils.aws.aws_responses import MessageConversion
 
 
 class ProxyListenerIAM(ProxyListener):
     def forward_request(self, method, path, data, headers):
         if method == 'POST' and path == '/':
-            data = self._reset_account_id(data)
+            data = MessageConversion._reset_account_id(data)
             return Request(data=data, headers=headers, method=method)
 
         return True
 
     def return_response(self, method, path, data, headers, response):
         if response.content:
             # fix hardcoded account ID in ARNs returned from this API
-            self._fix_account_id(response)
+            MessageConversion._fix_account_id(response)
             # fix dates returned from this API (fixes an issue with Terraform)
-            self._fix_date_format(response)
-            # fix error codes
-            self._fix_error_codes(method, data, response)
+            MessageConversion._fix_date_format(response)
```

Comment:
I think we're missing `MessageConversion._fix_error_codes(method, data, response)` here.